### DISCUSSION TITLE
Minecart doesn't break on cactus

### DISCUSSION THREAD

**Discussion by EnricoMariaUnibo** [TownyAdvanced/Towny#4832] [Q&A] [answered]
Morning,

Just like the title says when a minecart bumps into a cactus but the chunk is a town plot, this doesn't break.
Is there some kind of configuration i could use to prevent this from happening?
I already removed the minecart entity from the protected entities, but that doesn't resolve the problem.

Thank you!

**Comment by LlmDl** [comment, marked as answer]
This was fixed a while back you probably just have to update. If you are up to date file a bug report via Issues-> New Issue.
